code/game/machinery/computer/arcade.dm

```diff
@@ -50,6 +50,9 @@
 		qdel(src)
 
 /obj/machinery/computer/arcade/proc/prizevend()
+	if(prob(prob(prob(1)))) //1 in a million
```

Comment:
prob(100) 1 in 1
prob(10) 1 in 10
prob(1) 1 in 100
prob(0.1) 1 in 1000
prob(0.01) 1 in 10,000
prob(0.001) 1 in 100,000
prob(0.0001) 1 in 1,000,000

Rough decimal placement not withstanding